src/test/kotlin/coverosR3z/authentication/UserManagementTests.kt

```diff
@@ -0,0 +1,32 @@
+package coverosR3z.authentication
+
+import coverosR3z.domainobjects.Employee
+import coverosR3z.persistence.PureMemoryDatabase
+import org.junit.Test
+import coverosR3z.createTimeEntryPreDatabase
+import org.junit.Assert.assertEquals
+
+class UserManagementTests {
+    /**
+     * When a user is registered, they must supply an employee identifier provided by some
+     * administrator. We cannot allow a user to simply create their corresponding employee--
+     * they must be invited into the org, with some prior setup.
+     */
+
```

Comment:
By convention, method comments shouldn't have whitespace between them and the method